addons/core/translations/en-us.yaml

```diff
@@ -77,10 +77,12 @@ onboarding:
       desktop: Using Boundary Desktop
       developers: Developers
       controller: Controller URL
+      role: We have created your scopes and target
```

Comment:
This key/value mapping doesn't make sense to me.  Why is the key "role"?

Reply:
Updated to `creation-success`. Even that seems not exactly right but more accurate than "role".